label_studio/templates/base.html

```diff
@@ -98,26 +98,86 @@
 </template>
 
 <script id="app-settings" nonce="{{request.csp_nonce}}">
+
+  var __customHotkeys = {{ user.custom_hotkeys|json_dumps_ensure_ascii|safe }};
```

Comment:
why not creating `mergedEditorKeymap` in Django code instead of custom inline js ?

Reply:
Django doesn't know much about the defaults for the keys, and we don't want to have two sets of defaults, one on the backend, and another on the frontend, all of the definitions and processing is done at the frontend level, this follows DRY principle.